examples/typescript-react/readme.md

```diff
@@ -0,0 +1,80 @@
+
+> React is a JavaScript library for creating user interfaces. Its core principles are declarative code, efficiency, and flexibility. Simply specify what your component looks like and React will keep it up-to-date when the underlying data changes.
+
+> _[React - facebook.github.io/react](http://facebook.github.io/react)_
+
+## Learning TypeScript
+
+The [TypeScript website](http://typescriptlang.org) is a great resource for getting started.
+
+Here are some links you may find helpful:
+
+* [Tutorial](http://www.typescriptlang.org/Tutorial)
+* [Code Playground](http://www.typescriptlang.org/Playground)
+* [Documentation](http://typescript.codeplex.com/documentation)
+* [Applications built with TypeScript](http://www.typescriptlang.org/Samples)
+* [Blog](http://blogs.msdn.com/b/typescript)
+* [Source Code](http://typescript.codeplex.com/sourcecontrol/latest#README.txt)
+
+Articles and guides from the community:
+
+* [Thoughts on TypeScript](http://www.nczonline.net/blog/2012/10/04/thoughts-on-typescript)
+* [ScreenCast - Why I Like TypeScript](http://www.leebrimelow.com/why-i-like-typescripts)
+
+Get help from other TypeScript users:
+
+* [TypeScript on StackOverflow](http://stackoverflow.com/questions/tagged/typescript)
+* [Forums](http://typescript.codeplex.com/discussions)
+* [TypeScript on Twitter](http://twitter.com/typescriptlang)
+
+_If you have other helpful links to share, or find any of the links above no longer work, please [let us know](https://github.com/tastejs/todomvc/issues)._
+
+## Learning React
+
+The [React getting started documentation](http://facebook.github.io/react/docs/getting-started.html) is a great way to get started.
+
+Here are some links you may find helpful:
+
+* [Documentation](http://facebook.github.io/react/docs/getting-started.html)
+* [API Reference](http://facebook.github.io/react/docs/reference.html)
+* [Blog](http://facebook.github.io/react/blog/)
+* [React on GitHub](https://github.com/facebook/react)
+* [Support](http://facebook.github.io/react/support.html)
+
+Articles and guides from the community:
+
+* [How is Facebook's React JavaScript library](http://www.quora.com/React-JS-Library/How-is-Facebooks-React-JavaScript-library)
+* [React: Under the hood](http://www.quora.com/Pete-Hunt/Posts/React-Under-the-Hood)
+
+Get help from other React users:
+
+* [React on StackOverflow](http://stackoverflow.com/questions/tagged/reactjs)
+* [Discussion Forum](https://discuss.reactjs.org/)
+
+_If you have other helpful links to share, or find any of the links above no longer work, please [let us know](https://github.com/tastejs/todomvc/issues)._
+
+
+
+## Running
+
+A standalone TypeScript compiler is available on NPM.
+
+	npm install typescript
+
+To compile the TypeScript in this project:
+
+	# from examples/typescript-react
+	$ tsc -p js
+
+To be able to run the output JS files in the browser:
+
+	# from examples/typescript-react
+	$ npm install -g browserify
+	$ browserify js/app.js -o js/bundle.js
```

Comment:
Instead of all the global installations just add an `npm run compile` script (that's consistent with how we've been doing other examples lately)